src/test/analyzer_test.ts

```diff
@@ -683,6 +698,35 @@ var DuplicateNamespace = {};
           ['cyclic-a.html', 'root.html', 'subdir/root-in-subdir.html']
               .sort(), );
 
+      assert.deepEqual(
+          Array.from(pckage.getByKind('document'))
+              .filter((d) => !d.isInline)
+              .map((d) => d.url)
+              .sort(),
+          [
+            'cyclic-a.html',
+            'cyclic-b.html',
+            'root.html',
+            'leaf.html',
+            'subdir/subdir-leaf.html',
+            'subdir/root-in-subdir.html'
```

Comment:
The key thing here is that bower_components/imported.html isn't there unless you explicitly ask for it.